project/Dependencies.scala

```diff
@@ -34,13 +34,13 @@ object Dependencies {
   val kamonPrometheus     = "io.kamon"                   %% "kamon-prometheus"          % kamonVersion
   val lmdbjava            = "org.lmdbjava"                % "lmdbjava"                  % "0.6.1"
   val logbackClassic      = "ch.qos.logback"              % "logback-classic"           % "1.2.3"
-  val monix               = "io.monix"                   %% "monix"                     % "3.0.0-RC1"
+  val monix               = "io.monix"                   %% "monix"                     % "3.0.0-RC2-d0feeba"
```

Comment:
Finally :heart:  Does this version of `monix` implement all the new `cats-effect` typeclass? proper `Bracket` etc?

Reply:
I'm not sure. What I am sure is that they depend on cats-core 1.0.0 and are binary-compatible in general with the rest of ecosystem. Otherwise the build would fail just as it did for Kyle recently.